### PR TITLE
Add documentation for Bioconductor RBioFormats

### PR DESCRIPTION
**Pull request by dgault**
This is a follow up to the request raised on forum thread https://forum.image.sc/t/possible-addition-to-https-www-openmicroscopy-org-bio-formats-downloads-rbioformats/79474

This PR adds 2 new pages, adding Bioconductor to the Users list (under libraries and sciprting applications) and also a page on `Using Bio-Formats in R` under the developer section.

### CODE REVIEW

File: sphinx/developers/r-dev.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      install.packages("BiocManager")
+
+    # The following initializes usage of Bioc devel
+    BiocManager::install(version='devel')
+
+    BiocManager::install("RBioFormats")
+
+
+    # To view documentation for the version of this package installed in your system
+    browseVignettes("RBioFormats")
```
Comment (sbesson) [inline sphinx/developers/r-dev.rst:26]: Possibly the dev section could include a reference to https://bioconductor.org/packages/devel/bioc/vignettes/RBioFormats/inst/doc/RBioFormats.html which is the API documentation

**Review by sbesson — COMMENTED** [review]
Thanks @dgault, the addition of two separate pages (user & dev) makes sense to me and follows the established pattern for e.g. MATLAB and other applications - see https://bio-formats--295.org.readthedocs.build/en/295/users/index.html and https://bio-formats--295.org.readthedocs.build/en/295/developers/index.html.

My primary question is whether the user page should be called `Bioconductor` under `Libraries and scripting applications` or `R` under `Numerical data processing applications` e.g. by symmetry with MATLAB or Octave.

**Comment by dgault** [comment]
I can see either option making sense for the user page, happy to move it to `R` under `Numerical data processing applications` if that is preferred.

**Comment by melissalinkert** [comment]
Maybe worth adding a statement to the Bioconductor user page similar to what's at the top of the dev page (so it's clearer that this isn't something OME develops). Otherwise looks fine to me, I don't have strong opinions on which section to use for the user page.

File: sphinx/developers/r-dev.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+For further details and examples of how to use the package, please see the `API documentation<https://bioconductor.org/packages/devel/bioc/vignettes/RBioFormats/inst/doc/RBioFormats.html>`_
```
Comment (sbesson) [inline sphinx/developers/r-dev.rst:28]:
```suggestion
For further details and examples of how to use the package, please see the `API documentation <https://bioconductor.org/packages/devel/bioc/vignettes/RBioFormats/inst/doc/RBioFormats.html>`_ 
```

Looks like you need an extra space to get this rendered as an hyperlink.

**Review by sbesson — CHANGES_REQUESTED** [review]
One rendering issue. Otherwise looks like good to me

**Review by sbesson — APPROVED** [review]

**Pull request merged**